browser/src/docdispatcher.ts

```diff
@@ -91,6 +91,9 @@ class Dispatcher {
 		this.actionsMap['remotelink'] = function () {
 			app.map.fire('postMessage', { msgId: 'UI_PickLink' });
 		};
+		this.actionsMap['remoteaicontent'] = function () {
+			app.map.fire('postMessage', { msgId: 'UI_InsertAIContent' });
```

Comment:
so how does this work? What is the expected response to this message?

We'll need to update the documentation afterwards.

Reply:
We don't expect any response, the integrator can handle it with whatever they want. Our integration is inserting text for example: https://github.com/nextcloud/richdocuments/pull/4333

@pedropintosilva The smart picker is also not documented.

Reply:
it's on my list of things to document.